tox.ini

```diff
@@ -27,3 +27,10 @@ commands =
 commands =
     python --version
     nosetests {posargs}
+
+[testenv:flake8]
+description = "Code quality inspection using flake8"
```

Comment:
Wow, TIL!  Where does tox show this description?

Reply:
When you run `tox -va`. It's useful when your environments start to build up and you don't want to go digging through the source file just to remember what the commands for the project you're working on are. 😄